File: beginner/pagina11/2850/2850.cpp
#include <iostream>
using namespace std;

int main(){
    string input;

    while(getline(cin, input)){
        if(input == "esquerda"){ cout << "ingles" << endl; }
        if(input == "direita"){ cout << "frances" << endl; }
        if(input == "nenhuma"){ cout << "portugues" << endl; }
        if(input == "as duas"){ cout << "caiu" << endl; }
    }
    return 0;
}

File: beginner/pagina11/2850/2850.rs
use std::io;

fn main(){
    let mut input = String::new();

    while io:;stdin().read_line(&mut input).unwrap() > 0 {

        let mut output = String::new();

        if input.trim() == "esquerda" {
            output = "ingles".to_string();
        }

        else if input.trim() == "direita" {
            output = "frances".to_string();
        }

        else if input.trim() == "nenhuma" {
            output = "portugues".to_string();
        }

        else if input.trim() == "as duas" {
            output = "caiu".to_string();
        }

        println!("{}", output);
        input.clear();
    }
}
